sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -967,6 +1381,35 @@ public virtual Response CreateOrUpdateUser(string userId, RequestContent content
         /// <param name="fromTransactionId"> Specify the first transaction ID in a range. </param>
         /// <param name="toTransactionId"> Specify the last transaction ID in a range. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GetLedgerEntriesAsync and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new ConfidentialLedgerClient(endpoint, credential);
+        /// 
+        /// Response response = await client.GetLedgerEntriesAsync();
+        /// 
+        /// JsonElement result = JsonDocument.Parse(GetContentFromResponse(response)).RootElement;
+        /// Console.WriteLine(result.GetProperty("state").ToString());
+        /// Console.WriteLine(result.GetProperty("entries").Item[0].GetProperty("contents").ToString());
+        /// ]]></code>
+        /// This sample shows how to call GetLedgerEntriesAsync with all parameters, and how to parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new ConfidentialLedgerClient(endpoint, credential);
+        /// 
+        /// Response response = await client.GetLedgerEntriesAsync("<subLedgerId>", "<fromTransactionId>", "<toTransactionId>");
```

Comment:
`GetLedgerEntries` is a pageable operation.  The example should show how to use the `Pageable<BinaryData>` return value and parse the result from the `BinaryData` item.

Reply:
Fixed. Adding support for pageable operation